pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotDatabaseRestletResource.java

```diff
@@ -59,4 +67,91 @@ public class PinotDatabaseRestletResource {
   public List<String> listDatabaseNames() {
     return _pinotHelixResourceManager.getDatabaseNames();
   }
+
+  @DELETE
+  @Produces(MediaType.APPLICATION_JSON)
+  @Path("/databases/{databaseName}")
+  @Authorize(targetType = TargetType.CLUSTER, action = Actions.Cluster.DELETE_DATABASE)
+  @ApiOperation(value = "Delete all tables in given database name", notes = "Delete all tables in given database name")
+  public DeleteDatabaseResponse deleteTablesInDatabase(
+      @ApiParam(value = "Database name", required = true) @PathParam("databaseName") String databaseName,
+      @ApiParam(value = "Run in dryRun mode initially to know the list of tables that will be deleted in actual run. "
+          + "No tables will be deleted when dryRun=true", required = true)
+      @QueryParam("dryRun") boolean dryRun) {
```

Comment:
Can we define the default value to be `true`? It would be great if by default it is `dryRun` so the UI will mark it by default and users would need to actively turn it off.

Reply:
Agree, made the change 👍